pkg/server/plugin/nodeattestor/aws/ec2client.go

```diff
@@ -0,0 +1,10 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+// EC2Client interface describing used aws ec2client functions, useful for mocking
+type EC2Client interface {
+	DescribeInstances(input *ec2.DescribeInstancesInput) (*ec2.DescribeInstancesOutput, error)
```

Comment:
nit: this might be a good opportunity for us to switch over to `DescribeInstancesWithContext` to support cancellation.